lib/Article.js

```diff
@@ -41,7 +41,7 @@ export default React.createClass({
           </time>
         </header>
         <article className="article">
-          <h2 className="article__title">
+          <h2 className="article__contentTitle">
```

Comment:
Had to change this class name so it wouldnt affect the article previews titles on the main page